docs/standard_transaction.md

```diff
@@ -0,0 +1,147 @@
+---
+id: standard_transaction
+title: 6 - The Standard Transaction
+---
+
+You should now be well versed in a number of ways to lock up a coin using a Chialisp puzzle.  We have all the tools we need now to talk about the standard transaction format on the Chia network.
+
+Before you go through this section, it may be worth it to check out this [blog post](https://www.chia.net/2021/05/27/Agrgregated-Sigs-Taproot-Graftroot.html) by Bram Cohen on why the standard transaction is the way it is.
+
+## Pay to "Delegated Puzzle" or "Hidden Puzzle"
+
+If you remember from [part 2](/docs/coins_spends_and_wallets) we created a puzzle that paid to a "delegated puzzle": a puzzle that allows the solver to pass in a puzzle and solution to create their own conditions for the output.  This is one half of the functionality we want our standard transaction to have.
+
+However, we also want the ability to pre-commit to a puzzle without revealing it, and let anybody with the knowledge of the "hidden" puzzle spend it.
+
+But how do we pre-commit to this hidden puzzle?  We can curry it in, but if we perform the delegated spend case we will have to reveal the full puzzle including the curried in hidden puzzle and it will no longer be hidden.  We can't lock up a coin with the same puzzle anymore, or else people will be able to tell that the puzzle hash is the same and spend it without our consent.  Our delegated spend might not even make it to the network, a malicious node can just deny our transaction after seeing it and then publish the hidden spend case on their own.
+
+We can attempt to solve this by hashing the hidden puzzle.  This has some similar problems.  If you spend the hidden case even once, people can see any identical puzzle hashes later and spend them without your consent.  Furthermore, many people may try to use the same hidden puzzle.  If anyone reveals it, all coins locked up with that same puzzle can also be identified and spent.  We need the puzzle to be hidden, but also have some entropy that keeps it unique to us.
+
+The solution that the standard transaction uses is to derive a new private key from a) the hidden puzzle and b) the public key that can sign for the delegated spend case:
```

Comment:
Maybe expand this sentence. Talk about the advantages of hiding it in a piece of information that is already there and only derivable by someone with the prior information up front.